report_thesis/src/sections/problem_definition.tex

```diff
@@ -0,0 +1,54 @@
+We showed that advanced ensemble methods, such as \gls{gbr}, and deep learning models, such as \gls{ann}, have the potential to outperform the current MOC pipeline.
+Methods are selected based on their promise in handling high-dimensional, non-linear data. Ideally, the selected methods should also be feasible for small datasets, a common scenario in \gls{libs} analyses. 
+In order to address the aforementioned challenges, we propose to explore advanced ensemble methods and deep learning models, which have shown promise in handling high-dimensional, non-linear data.
+
+It is necessary to establish metrics to evaluate the performance of the models.
+In \cite{p9_paper}, we proposed to use the \gls{rmse} as a proxy for accuracy.
+
+\gls{rmse} is given by:
+
+\begin{equation}
+    RMSE = \sqrt{\frac{1}{n} \sum_{i=1}^{n} (y_i - \hat{y}_i)^2}
+\end{equation}
+
+where $y_i$ represents the actual values, $\hat{y}_i$ the predicted values, and $n$ the number of observations.
+
+To address robustness, we propose considering the standard deviation of prediction errors across each oxide and test instance, defined as:
+
+\begin{equation}
+    \sigma_{error} = \sqrt{\frac{1}{n-1} \sum_{i=1}^{n} (e_i - \bar{e})^2}
+\end{equation}
+
+where $e_i = y_i - \hat{y}_i$ and $\bar{e}$ is the mean error.
+
+In order to narrow down the scope of our research, we set the following constraints:
+\begin{itemize}
+    \item Prioritize normalization across individual spectrometers' wavelength ranges over full-spectrum normalization.
+    \item Focus on techniques proven effective for non-linear, high-dimensional data, even outside the \gls{libs} context.
+    \item Ensure methods are feasible for small datasets.
+\end{itemize}
+
+In \cite{p9_paper}, we used both full-spectrum normalization (Norm 1) and normalization across individual spectrometers' wavelength ranges (Norm 3).
+However, in this work, we opt for normalizing across individual spectrometers' wavelength ranges.
+This decision is guided by the operational parameters of SuperCam\cite{andersonPostLandingMajorElements2022}, where they do not normalize across the entire spectrum, but rather across individual spectrometers' wavelength ranges.
+In order to ensure the future applicability of our methods, we follow the same normalization approach.
+
+Our methodologies are selected to ensure compatibility with small datasets, a common scenario in \gls{libs} analyses.
+This consideration informs our preference for cross-validation techniques, which offer robust performance assessments without necessitating large data volumes.
+Additionally, the computational complexity of models is deemed a secondary concern, given the scale of data typically involved in \gls{libs} studies, allowing for a broader exploration of sophisticated, potentially more computationally intensive algorithms.
```

Comment:
This seems to come out of the blue, however I realize that it's supposed to elaborate on the constraints listed above. This goes back to my previous comment - wouldn't it be best to elaborate on each constraints in-situ instead of further down? This would make it much easier to read, and we wouldn't be jumping back and forth so much. I propose listing a constraint, then immediately elaborate on the reasoning for this constraint before listing the next constraint.

Reply:
I don't see the comment you reference, but I've decided to simply delete this part. It's more relevant in the methodology section.